source/content/php-runtime-generation-2.md

```diff
@@ -0,0 +1,137 @@
+---
+title: PHP Runtime Generation 2 (Beta)
+description: The second generation of Pantheon's PHP runtime is now available with added security and updated extensions.
```

Comment:
Docs content is generally assumed to be evergreen so "now" is somewhat meaningless.
```suggestion
description: The second generation of Pantheon's PHP runtime with added security and updated extensions.
```